pkg/slack/command.go

```diff
@@ -0,0 +1,244 @@
+package slack
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/oursky/github-actions-manager/pkg/utils/array"
+	"github.com/slack-go/slack"
+	"go.uber.org/zap"
+)
+
+type Argument struct {
+	name        string
+	required    bool
+	acceptsMany bool
+	description string
+}
+
+func NewArgument(name string, required bool, acceptsMany bool, description string) Argument {
+	return Argument{name: name, required: required, acceptsMany: acceptsMany, description: description}
+}
+
+type Command struct {
+	trigger     string
+	arguments   []Argument
+	description string
+	execute     func(env *CLI, args []string) CLIResult
+}
+
+type CLIResult struct {
+	printToChannel bool
+	message        string
+}
+
+func NewCLIResult(printToChannel bool, message string) CLIResult {
+	return CLIResult{printToChannel: printToChannel, message: message}
+}
+
+type CLI struct {
+	commands  *[]Command
+	app       *App
+	ctx       context.Context
+	channelID string
+	logger    *zap.Logger
+}
+
+func (c Command) String() string {
+	output := fmt.Sprintf("`%s`: %s", c.trigger, c.description)
+	output += fmt.Sprintf("\nUsage of `%s`: `%s", c.trigger, c.trigger)
+	for _, arg := range c.arguments {
+		argname := arg.name
+		if arg.acceptsMany {
+			argname += "..."
+		}
+		if arg.required {
+			output += " " + argname
+		} else {
+			output += " [" + argname + "]"
+		}
+	}
+	output += "`"
```

Comment:
This backtick looks like a mistake.

Reply:
It matches the backtick on line 50.
But you are right, this is bad code.
I'll fix it